CMakeLists.txt

```diff
@@ -339,7 +339,7 @@ function(protobuf_generate_grpc_cpp)
              "${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}_mock.grpc.pb.h"
              "${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}.pb.cc"
              "${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}.pb.h"
-      COMMAND $<TARGET_FILE:${_gRPC_PROTOBUF_PROTOC}>
+      COMMAND ${_gRPC_PROTOBUF_PROTOC}
```

Comment:
When running with cmake . -DgRPC_PROTOBUF_PROVIDER=package -DgRPC_ZLIB_PROVIDER=package -DgRPC_SSL_PROVIDER=package
I get

CMake Error at CMakeLists.txt:336 (add_custom_command):
  Error evaluating generator expression:

    $<TARGET_FILE:/usr/bin/protoc>

When {_gRPC_PROTOBUF_PROTOC} point to absolute path then prefixing it with anything doesn't make sense. I'm not sure what was the original intention of using TARGET_FILE here, I guess it accidentally pointed to the same directory as PROTOBUF_ROOT_DIR